crates/serde/src/storage.rs

```diff
@@ -26,13 +43,31 @@ use serde::{Deserialize, Deserializer, Serialize};
 ///
 /// The contained [B256] and From implementation for String are used to preserve the input and
 /// implement this behavior from geth.
-#[derive(Clone, Copy, Debug, Default, PartialEq, Eq, Serialize, Deserialize)]
-#[serde(from = "U256", into = "String")]
-pub struct JsonStorageKey(pub B256);
+#[derive(Clone, Copy, Debug, Default, PartialEq, Eq, Serialize)]
+#[serde(into = "String")]
```

Comment:
@Rjected isn't all of this now just serde(untagged) ?

Reply:
yeah I think so